gspread/models.py

```diff
@@ -12,8 +12,12 @@
 from collections import defaultdict
 from itertools import chain
 
-from xml.etree import ElementTree
-from xml.etree.ElementTree import Element, SubElement
+try:
+    from lxml.etree import ElementTree
```

Comment:
For lxml 3.4.4 this imports a wrong object:

~~~python
>>> from lxml.etree import ElementTree
>>> ElementTree.__doc__
'ElementTree(element=None, file=None, parser=None)\n\n    ElementTree wrapper class.\n    '
>>> ElementTree.tostring
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'builtin_function_or_method' object has no attribute 'tostring'
~~~

Wherein `etree` is actual `ElementTree` implementation.

~~~python
>>> from lxml import etree
>>> etree.__doc__
'The ``lxml.etree`` module implements the extended ElementTree API\nfor XML.\n'
~~~

This error is hidden since the `ImportError` exception in a line below leads to import of the stock `ElementTree`.

The [lxml docs](http://lxml.de/compatibility.html) suggest this way of the compatibility import:

~~~python
from lxml import etree as ElementTree
~~~